deployment/terraform/batch.tf

```diff
@@ -64,8 +64,6 @@ data "template_file" "default_job_definition" {
     google_server_side_api_key = "${var.google_server_side_api_key}"
 
     aws_region = "${var.aws_region}"
-
-    rollbar_client_side_access_token = "${var.rollbar_client_side_access_token}"
```

Comment:
Just realized, we should be threading through the `rollbar_server_side_access_token` here, as well as in the job definition so that exceptions in Batch jobs are surfaced to Rollbar.  

Also happy with splitting this out into a separate issue.

Reply:
Added that in af2c914.